sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -510,7 +513,19 @@ def input_elements(
             return
           if self._exception:
             raise self._exception from None
+          current_time = time.time()
+          if next_waiting_log_time <= current_time:
+            # If at the same time another instruction is waiting on input queue
+            # to become available, it is a sign of inefficiency in data plane.
+            _LOGGER.info(
+                'Waiting to receive elements in input queue '
+                'for instruction: %s for %.2f seconds.',
+                instruction_id,
+                current_time - start_time)
```

Comment:
can we add `log_interval_sec` in the log and mention this frequency? 
Was this tested with Dataflow jobs?

Reply:
> can we add log_interval_sec in the log and mention this frequency?

done